github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -129,6 +130,67 @@ periodics:
         cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
         ./hack/bump-centos-version.sh &&
         GIT_ASKPASS=../project-infra/hack/git-askpass.sh ../project-infra/hack/git-pr.sh -c "PHASES=linux BYPASS_PMAN_CHANGE_CHECK=true ./publish.sh" -r kubevirtci -b bump-centos-stream -T main -p $(pwd) -s "Automatic bump of CentOS Stream to latest"
+        # For passing centos image tag to dependent (s390x) prow job
+        SHORT_SHA=$(git rev-parse --short HEAD) &&
+        image_tag=$(cat cluster-provision/k8s/base-image | cut -d ':' -f 2) &&
+        echo "$image_tag" > amd64-centos9-$SHORT_SHA &&
+        gsutil cp ./amd64-centos9-$SHORT_SHA gs://kubevirt-prow/release/kubevirt/kubevirtci/amd64-centos9-$SHORT_SHA
+      # docker-in-docker needs privileged mode
+      env:
+      - name: GIMME_GO_VERSION
+        value: "1.22.5"
+      securityContext:
+        privileged: true
+      resources:
+        requests:
+          memory: "29Gi"
+- name: periodic-kubevirtci-bump-centos-base-s390x
+  cron: "0 4 * * 2"
```

Comment:
Does it make sense to trigger this job after the x86 job instead of triggering them at the same time?

Reply:
@brianmcarey I actually kept it that way to first run x86 followed by s390x. But later thought that it is not necessary, as we are waiting on x86 to be finished (write amd64-centos9-$SHORT_SHA) before triggering publish.sh in s390x. As generally x86 job finishes within 30mins max, though this job waits for that much time, it can be continued immediately.
May be we can keep a gap of 15mins or so if we think that way we can reduce pressure on cluster resource usage, due to this continuous polling for the file existance. WDYT?

Reply:
I would just start this job an hour after it - then we should be fairly sure that the tag will be there.

Reply:
Sure, will update this